pkg/routine/recurringtask_test.go

```diff
@@ -30,14 +30,13 @@ func (h *MockHandler) Do() {
 }
 
 func TestRecurringTask(t *testing.T) {
+	require := require.New(t)
 	h := &MockHandler{Count: 0}
 	ctx := context.Background()
 	ck := clock.NewMock()
 	task := routine.NewRecurringTask(h.Do, 100*time.Millisecond, routine.WithClock(ck))
 	task.Start(ctx)
 	ck.Add(600 * time.Millisecond)
 	task.Stop(ctx)
-	h.mu.RLock()
-	assert.True(t, h.Count >= 5)
-	h.mu.RUnlock()
```

Comment:
revert, it does not hurt to use `RLock()` here, no need to specifically delete this